components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/constants/AccountConstants.java

```diff
@@ -31,8 +31,8 @@
     public static final String FAILED_LOGIN_ATTEMPTS_PROPERTY = "account.lock.handler.On.Failure.Max.Attempts";
     public static final String LOGIN_FAIL_TIMEOUT_RATIO_PROPERTY = "account.lock.handler.login.fail.timeout.ratio";
 
-    public static final String EMAIL_TEMPLATE_TYPE_ACC_LOCKED = "account.locked";
-    public static final String EMAIL_TEMPLATE_TYPE_ACC_UNLOCKED = "account.unlocked";
+    public static final String EMAIL_TEMPLATE_TYPE_ACC_LOCKED = "accountlock";
+    public static final String EMAIL_TEMPLATE_TYPE_ACC_UNLOCKED = "accountunlock";
```

Comment:
When we are loading the email template from the registry, we are using the notification template type for generating the registry path. So that here it needs to get changed. I have made the changes to AccountDisableHandler as well.